packages/instantsearch.js/src/connectors/trending-items/__tests__/connectTrendingItems-test.ts

```diff
@@ -129,8 +129,9 @@ describe('connectTrendingItems', () => {
         new RecommendParameters().addTrendingItems({
           // @ts-expect-error
           $$id: widget.$$id,
-          facetName: undefined,
-          facetValue: undefined,
+          // v5 expects only string values for facetName and facetValue
+          facetName: undefined as unknown as string,
+          facetValue: undefined as unknown as string,
```

Comment:
https://github.com/algolia/api-clients-automation/pull/3355 after this is released we can amend to remove the cast. Still find for now as it's just a test though